simple/views/welcomePage.pug

```diff
@@ -0,0 +1,288 @@
+
+  body
+    // NAVIGATION BAR - need to fix for this page
+    nav.navbar.navbar-expand-sm.bg-success.bg-gradient.navbar-dark
+      .container-fluid
+        a.navbar-brand(href='welcomePage') Logo
+        |         
+        button.navbar-toggler(type='button', data-bs-toggle='collapse', data-bs-target='#collapsibleNavbar')
+          span.navbar-toggler-icon
+        |         
+        #collapsibleNavbar.collapse.navbar-collapse.justify-content-end
+          ul.navbar-nav
+            li.nav-item
+              a.nav-link(href='#') Attend
+            |                 
+            li.nav-item
+              a.nav-link(href='sponsorPage') Sponsor Us!
+            |                 
+            li.nav-item
+              a.nav-link(href='judgingCriteria') Judging Criteria
+            |                 
+            li.nav-item
+              a.nav-link(href='hardware') Hardware
+            |                 
+            li.nav-item
+              a.nav-link(href='pastEvents') Past Events
+            |                 
+            li.nav-item.dropdown
+              a.nav-link.dropdown-toggle(href='#', role='button', data-bs-toggle='dropdown') Dropdown
+              |                     
+              ul.dropdown-menu
+                li
+                  a.dropdown-item(href='/users') DB Data 
+                |                         
+                li
+                  a.dropdown-item(href='sponsor') Sponsors
+                |                         
+                li
+                  a.dropdown-item(href='#') Poll Results
+    #edit(contenteditable='false')
+      h1.display-5 Information
+      |     Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+      |     aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+      |     Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+      |     sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      |     
+      h1.display-5 Schedule
+      // SCHEDULE
+      .container.mt-4
+        ul.nav.nav-tabs.nav-justified
+          li.nav-item
+            a.nav-link.active(data-bs-toggle='tab', href='#one') 1st day
+          |             
+          li.nav-item
+            a.nav-link(data-bs-toggle='tab', href='#two') 2nd day
+          |             
+          li.nav-item
+            a.nav-link(data-bs-toggle='tab', href='#three') 3rd day
+        // Tab panes
+        .tab-content
+          #one.container.tab-pane.active
+            br
+            |                 
+            h3 One
+            |                 
+            p
+              .container.mt-3
+                table.table.table-striped
+                  tbody
+                    tr
+                      td 4:00-5:00
```

Comment:
How simple will this be to abstract later? How easy will it be for a user to change?